### ISSUE TITLE
Initial solution for data for F&A and Top Stories components

### ISSUE THREAD

**Issue by jamesdonoh** [bbc/simorgh#5765]
**Is your feature request related to a problem? Please describe.**
We need to populate the F&A and Top Stories components with data from the front page of each service. How this data is selected seems to vary depending on the service.

We're proposing to start with a basic implementation to demonstrate the components working end-to-end with real data for (at least) one service.

In parallel to this task we will continue investigating how these components are populated across different services and implement a comprehensive mapping later on as a separate issue.

**Describe the solution you'd like**
Create a simple solution that works for 1 or more services:
- Use the front page data response (e.g. https://www.bbc.com/nepali.json, http://localhost:7080/nepali.json)
- Delete fixture data `featuresAnalysis.json`, `featuresAnalysisRtl.json`, `topStories.json`, `topStoriesRtl.json`.
- Do some investigation on how this is currently implemented for Most Read, Radio Schedules, and present a proposed solution to the team channel before implementing.
- Implement a simple mapping to pull data from the correct group in the front page response (e.g. `content.groups[5].items`)
- Use this to populate the F&A and Top Stories components
- Add integration tests to make sure the F&A and Top Stories component columns are on the page
- Graceful degradation if the components cannot be rendered for any reason. i.e. The main page should not be affected, if the data fails to render, and the component should not be rendered.

<img width="485" alt="Screenshot 2020-03-04 at 16 40 14" src="https://user-images.githubusercontent.com/11161809/75901708-db607c80-5e36-11ea-9b08-81d9744e5f84.png">

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

**Comment by mykeels** [comment]
The `Most Read` [implementation](https://github.com/bbc/simorgh/blob/a1642893afb8febc2278d98c306dc56d0f15cb17/src/app/containers/MostRead/Canonical/index.jsx#L123) fetches its data from within the component. This approach will not work for our use-case, because multiple components depend on the data from a single source.

The `Radio Schedule` [implementation](https://github.com/bbc/simorgh/blob/a1642893afb8febc2278d98c306dc56d0f15cb17/src/app/containers/RadioSchedule/index.jsx#L23) works the same way too.

**Comment by chris-hinds** [comment]
Blocked on the data feed from the Ares team

**Comment by HarveyPeachey** [comment]
Closing due to duplicate of https://github.com/bbc/simorgh/issues/6252
